stable/datadog/templates/daemonset.yaml

```diff
@@ -133,7 +133,7 @@ spec:
           - name: DD_CLUSTER_AGENT_AUTH_TOKEN
             valueFrom:
               secretKeyRef:
-                name: {{ template "datadog.clusterAgent.fullname" . }}
+                name: {{ include "datadog.fullname" . }}-cluster-agent
```

Comment:
This needs to be updated 
https://github.com/helm/charts/pull/11903/commits/e8e3405a850ea2294f73d8557c4329409fed4280#diff-c5ed47d16bb168ac09be3d6cdf6b4abaR132